scripts/result_scanner/test/test_result_scanner.rb

```diff
@@ -0,0 +1,140 @@
+require 'minitest/autorun'
+require_relative '../result_scanner'
+
+# For now, you should just be able to `ruby test_result_scanner.rb`
+
+class TestResultScanner < MiniTest::Test
+
+  def setup
+    @last_run_path = "#{__dir__}/fixtures/last_run"
+    @current_run_path = "#{__dir__}/fixtures/current_run"
+    @last_json = ResultScanner.read_json("#{@last_run_path}/fake-site-1.json")
+    @current_json = ResultScanner.read_json("#{@current_run_path}/fake-site-1.json")
+    @last_results = ResultScanner.create_results("fake-site-1", @last_json)
+    @current_results = ResultScanner.create_results("fake-site-1", @current_json)
```

Comment:
Yes.  I've used FactoryGirl in the past.  CT needs a "real" test suite that can load deps for ruby (and js and python?). I'm not sure what that looks like (?). And once that happens these tests will require some more work.  They mostly exist as dev artifacts atm.